### PR TITLE
Switch to v2 Hydra checklist endpoint + logging

### PR DESCRIPTION
**Pull request by Allda**
The new v2 endpoint supports multiple checklists for a single project.
The new endpoint returns a different response body with multiple
checklists. This commit iterates over all checklists + all checks and
reports failed tests.

This pull request also contains logging changes. 

JIRA: ISV-1492

### CODE REVIEW

File: operator-pipeline-images/operatorcert/entrypoints/hydra_checklist.py [hunk 1]
```diff
@@ -81,7 +116,7 @@ def main() -> None:
     log_level = "INFO"
     if args.verbose:
         log_level = "DEBUG"
-    logging.basicConfig(level=log_level)
+    setup_logger(log_level)
```
[the remark this answers is not in the document]
Comment (Allda) [reply operator-pipeline-images/operatorcert/entrypoints/hydra_checklist.py]: Fixed

File: operator-pipeline-images/operatorcert/entrypoints/hydra_checklist.py [hunk 2]
```diff
@@ -34,13 +35,49 @@ def setup_argparser() -> Any:  # pragma: no cover
...
+    """
+    Call Hydra checklist API and check if all test passed
+
+    Args:
+        cert_project_id (str): Certification project identifier
+        hydra_url (str): Base Hydra API URL
+        ignore_publishing_checklist (bool): The flag that determines if the tool fail
+            in case a checklist contains failed tests. If the flag is set to True
+            and there is a failed test, the tool doesn't fail with error code
+            and only prints out failed tests.
```
[the remark this answers is not in the document]
Comment (Allda) [reply operator-pipeline-images/operatorcert/entrypoints/hydra_checklist.py]: Fixed

File: operator-pipeline-images/operatorcert/entrypoints/ocp_version_info.py [hunk 3]
```diff
@@ -22,19 +25,21 @@ def setup_argparser() -> argparse.ArgumentParser:  # pragma: no cover
...
     return parser
 
 
 def main() -> None:
-    logging.basicConfig(stream=sys.stdout, level="INFO", format="%(message)s")
 
     parser = setup_argparser()
     args = parser.parse_args()
+    log_level = "DEBUG" if args.verbose else "INFO"
+    setup_logger(log_level)
```
[the remark this answers is not in the document]
Comment (Allda) [reply operator-pipeline-images/operatorcert/entrypoints/ocp_version_info.py]: Fixed

**Review by amisstea — APPROVED** [review]

**Review by haripate — APPROVED** [review]
LGTM

**Pull request merged**